### DISCUSSION TITLE
'datacsysfontsize' is not changing the font size

### DISCUSSION THREAD

**Discussion by shahul31** [neperfepx/neper#501] [Q&A] [answered]
Hello,

Even after using this 'datacsysfontsize' option to change the font size of the scale, the font size remains unchanged. Please help!

**Comment by rquey** [comment, marked as answer]
Please provide an example.

**Reply by shahul31** [reply]
Please find the below example, 

`neper -V p50_1.sim  \
    -simstep 3 \
    -datanodecoo coo \
    -dataelt1drad 0.004 \
    -dataelt3dedgerad 0.0015 \
    -datanodecoofact 10 \
    -dataelt3dcol strain22 \
    -dataeltscaletitle "Strain 22" \
	-datacsysfontsize 0.1 \
	-showelt1d all \
    -cameraangle 13.5 \
    -imagesize 1000:1300 \
	-print test2_e22_full`
	
prints scale like 
![1_e22 - Copy](https://user-images.githubusercontent.com/67755529/177121728-31715d0c-341c-4a2a-9229-570158158085.png)

The default size is given as 1, so I tried to reduce the size. But increasing or decreasing the "datacsysfontsize" is not showing any effect on the printed results. This is not a big issue, but I am trying to run a large number of simulations and getting results, in this case correcting the scale every time is a problem, so I am trying to reduce the font size to view it better.

**Reply by rquey** [reply]
This is not what `-datacsysfontsize` does; it sets the font of the labels of the coordinate system.

As far as the scale is concerned, the font size cannot be modified, but the values can be chosen / formatted. In your case, I would use `-dataelt3dcol real:100*strain22 -dataeltscaletitle "Strain 22 [%]" -dataeltscale -0.40:0`, which produces the image below. Note that I used `-0.40` (with two decimal digits), which is how the format can be specified.

![image](https://user-images.githubusercontent.com/2659376/177124876-c31fb674-b29b-473e-b427-8f7c1b40c2ee.png)

With `-dataeltscale -0.4:-0.3:-0.2:-0.1:0`:
![image](https://user-images.githubusercontent.com/2659376/177126850-64c47a0d-bed8-4091-a7f0-fe565dbb17fd.png)

**Reply by shahul31** [reply]
Thank you very much for the clarification. This is helpful :)
